src/main/scala/beam/utils/csv/writers/BeamCsvScenarioWriter.scala

```diff
@@ -0,0 +1,43 @@
+package beam.utils.csv.writers
+
+import scala.language.implicitConversions
+
+import beam.utils.scenario.{HouseholdInfo, PersonInfo, VehicleInfo}
+import com.typesafe.scalalogging.LazyLogging
+
+object BeamCsvScenarioWriter extends LazyLogging {
```

Comment:
What are all these implicits about? Could you, please, provide an example of the usage new writers?

Reply:
Added to the PR description and also added as comments to the file